src/main/resources/templates/fragments/navbar.html

```diff
@@ -5,7 +5,7 @@
     const noFavourites = /*[[#{noFavourites}]]*/ '';
     const updateAvailable =  /*[[#{settings.updateAvailable}]]*/ '';
   </script>
-  <script src="js/githubVersion.js"></script>
+  <script th:if="${@shouldShow}" src="js/githubVersion.js"></script>
```

Comment:
I think not showing in main UI makes sense but not showing in settings menu seems wrong

 feel it should always be on

Reply:
ah, I just see that you didn't want it to be completely hidden